mobile/ios/Podfile.lock

```diff
@@ -13,6 +13,8 @@ PODS:
   - FMDB/standard (2.7.5)
   - image_picker_ios (0.0.1):
     - Flutter
+  - isar_flutter_libs (1.0.0):
```

Comment:
I believe this is an accident?
`isar` is not added as a package

Reply:
Yes, it was a poc left over of trying out the Isar package. I did a few implementation using sqlite and isar package to see which persistent storage would work well and Hive fits well in between of complexity and performance

Reply:
At some point I'll probably want to introduce isar for full offline support. But until then, we don't need it :-)